internal/pkg/cli/local_run.go

```diff
@@ -292,23 +297,51 @@ func (o *localRunOpts) Execute() error {
 	}
 	o.imageInfoList = append(o.imageInfoList, sidecarImageLocations...)
 
-	err = o.runPauseContainer(context.Background(), containerPorts)
-	if err != nil {
-		return err
-	}
+	errCh := make(chan error, 1)
+	// Handle containers when a termination signal is received.
+	go func() {
+		sig := o.NotifyInterrupt()
+		fmt.Println("Received signal:", sig)
+		o.mutex.Lock()
+		o.isContainerTermination = true
```

Comment:
Just to doublecheck: does it work like as soon as they press `ctrl+c` we stop https://github.com/aws/copilot-cli/pull/5200/files#diff-76917d2837bb1ae692cefe45c798014527fbd660bc5cc12674b3ba91c73a3630R315 and run `o.handleContainers()` for the cleanup process? Shall we use `context` here instead of a boolean var?